packages/sit-onyx/src/composables/useCustomValidity.ts

```diff
@@ -13,7 +15,12 @@ export type UseCustomValidityOptions = {
   /**
    * Component props as defined with `const props = defineProps()`
    */
-  props: CustomValidityProp & { modelValue?: unknown };
+  props: CustomValidityProp & {
+    modelValue?: unknown;
+    type?: (typeof TRANSLATED_INPUT_TYPES)[number] | string;
```

Comment:
```suggestion
    type?: InputType;
```

`| string` is more generic than the TRANSLATED_INPUT_TYPES so the resulting type will just be `string`.
We can use `InputType` instead